### PR TITLE
Remove utils.py functions for their dask/utils.py equivalents

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

**Comment by mrocklin** [comment]
Pushing a couple of things upstream in https://github.com/dask/dask/pull/5384

**Comment by TomAugspurger** [comment]
Merged the dask side.

Are we bumping the minimum required dask version in the next distributed release?

**Comment by mrocklin** [comment]
Let's let this simmer until the next distributed release, then merge and
bump version numbers.  That should create enough slop space I hope.

File: distributed/utils.py [hunk 1]
```diff
@@ -38,7 +37,14 @@
 from dask import istask
 
 # provide format_bytes here for backwards compatibility
-from dask.utils import format_bytes  # noqa
+from dask.utils import (  # noqa
+    format_bytes,
+    funcname,
+    format_time,
+    parse_bytes,
+    parse_timedelta,
```
[the remark this answers is not in the document]
Comment (mrocklin) [reply distributed/utils.py]: We use this with config values like `"timeout": "10 s"`.  I think it's generally useful across Dask subprojects.  Yes, it's probably only for projects that also depend on distributed, but I wouldn't mind keeping the `parse_foo` functions together.

**Comment by mrocklin** [comment]
I plan to merge this after the next release if that's ok with folks.

**Pull request merged**